src/main/resources/templates/showFoodArchive.html

```diff
@@ -0,0 +1,189 @@
+
+            </form>
+
+            <a href="/food" class="btn btn-primary mr-2 btn-sm">Show food <i class="fas fa-drumstick-bite"></i></a>
+
+
+        </div>
+
+        <table class="table table-sm">
+            <thead>
+            <tr>
+                <!--<th>Food ID</th>-->
+                <th>
+                    <a th:href="@{'/pageFood/'+ ${currentPage} + '?sortField=name&sortDir=' + ${reverseSortDir}}">Name </a>
+                </th>
+                <th>
+                    <a th:href="@{'/pageFood/'+ ${currentPage} + '?sortField=protein&sortDir=' + ${reverseSortDir}}">Protein </a>
+                </th>
+                <th>
+                    <a th:href="@{'/pageFood/'+ ${currentPage} + '?sortField=carbohydrates&sortDir=' + ${reverseSortDir}}">Carbohydrates </a>
+                </th>
+                <th>
+                    <a th:href="@{'/pageFood/'+ ${currentPage} + '?sortField=fat&sortDir=' + ${reverseSortDir}}">Fat </a>
+                </th>
+                <th>
+                    KiloJoule
+<!--                <a th:href="@{'/page/'+ ${currentPage} + '?sortField=energy_kilojoule &sortDir=' + ${reverseSortDir}}">KiloJoule </a>-->
+                </th>
+                <th>
+                    Kcal
+<!--                <a th:href="@{'/page/'+ ${currentPage} + '?sortField=energy_kcal &sortDir=' + ${reverseSortDir}}">Kcal </a>-->
+                </th>
+                <th></th>
+
+            </tr>
+            </thead>
+            <tbody>
+            <tr th:each="food : ${listFood}">
+                <!--<td th:text="${food.id}">Food ID</td>-->
+                <th:block th:if="${food.archived == true}">
+                    <td><i class="fas fa-drumstick-bite"></i> <span th:text="${food.name}"></span></td>
+                    <td th:text="${food.protein}">Protein</td>
+                    <td th:text="${food.carbohydrates}">Carbohydrates</td>
+                    <td th:text="${food.fat}">Fat</td>
+                    <td th:text="${food.energy_kilojoule}">Kilojoule</td>
+                    <td th:text="${food.energy_kcal}">Kcal</td>
+                    <td class="actionButtons">
+                        <a th:href="@{/updateFood/{id}(id=${food.id})}" class="btn btn-outline-dark mr-2 btn-sm"><i class="fas fa-pen"></i> Edit</a>
+                        <!--<a th:href="@{/deleteFood/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm" th:attr="onclick=|return confirm_delete('${food.name}')|"><i class="fas fa-trash-alt"></i> Delete</a>-->
+                        <a th:if="${food.fk_user_id == 0}" th:href="@{/archiveFood/false/food/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm"><i class="fas fa-undo-alt"></i> Restore</a>
+                        <a th:if="${food.fk_user_id != 0}" th:href="@{/archiveFood/false/pfood/{id}(id=${food.id})}" class="btn btn-outline-danger mr-2 btn-sm"><i class="fas fa-undo-alt"></i> Restore</a>
```

Comment:
`class="btn btn-outline-danger mr-2 btn-sm"` should be `class="btn btn-outline-primary mr-2 btn-sm"` instead